app/repositories/ActiveSessionRepository.scala

```diff
@@ -49,12 +49,49 @@ class ActiveSessionRepositoryImpl @Inject()(mongo: MongoDriver,
   private val lastUpdatedIndex = Index(
     key = Seq("updatedAt" -> IndexType.Ascending),
     name = Some("session-updated-at-index"),
+    unique = false,
+    background = false,
+    sparse = false,
+    expireAfterSeconds = None,
```

Comment:
Do we need to check if we should provide this one? We used to, and still down (as below) pass this in as options.

Reply:
@adamconder The Method that allows the default values is now deprecated so we can't use it

Reply:
@adamconder Who would I check with to see what the expiry time should be? The value I've set it to is the same value as it was defaulting to, but I agree, it's probably worth setting to a real expiry time